### PR TITLE
Address deleted files

### PR DESCRIPTION
**Pull request by kbarnhart**
@SiccarPoint this may be slightly tricker than I expected given that @mcflugen merged his changes to the examples folder (#835) at the same time, I merged from master a few times, your branch merged from master a few times, and changes have happened since. 

I've learned a lot about searching the git graph, cherry picking, and reverting... And at the moment I think what makes the most sense is reverting the single reverting commit:

```
git revert adf20d52c44c8e006f2b4ac239a7af3e4b39fdaf
```

And then carefully checking that this gets us back where we expect. I'll do this with a careful git diff between my current branch commit, the master this branch was merged into (f781650fc456a136a0717c1a674446ec79f982e3), and the commit when this branch was merged (
fe27cc65d3dee6779461b86bed180132b971be7a).  

@mcflugen if you have a better idea idea, please let me know!

@SiccarPoint I anticipate I'll finish doing this on Monday. Is that timeline oK with you?

### CODE REVIEW

**Comment by SiccarPoint** [comment]
That sounds awesome to me. Thanks for taking a look!

**Comment by kbarnhart** [comment]
@SiccarPoint @mcflugen this is now ready for review. 

I tried a couple of other options for fixing this (hence some force pushes) but what I originally thought was a feasible way was the only feasible way. 

I make a file-by-file comparison having only made the single reversion and I think everything is back to being correct. The number of changed lines did not add up perfectly, but was close (e.g. 1,497 additions and 5,709 deletions vs 1,481 deletions and 5,913 addition). 

Once I did that check I did a small amount of cleanup (including updating to the new origin methods). I also had to change a small number of things in one of Dan's files. I'll make a comment in the code so you know where to look. I tried to describe why I thought my change made sense as a doctoring comment, so hopefully it will make sense to you.

File: landlab/components/lake_fill/lake_fill_barnes.py [hunk 1]
```diff
@@ -0,0 +1,2182 @@
...
+        ...                            ignore_overfill=True)
+        ... except ValueError:
+        ...     print('Oops!')
+        Oops!
+
+        The component is completely happy with irregular grids:
+
+        >>> from landlab import HexModelGrid, FieldError
+        >>> hmg = HexModelGrid(5, 4, dx=2.)
+        >>> z_hex = hmg.add_zeros('node', 't
```
Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: @SiccarPoint I added this print out for debugging.

Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: I also added this comment

Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: The 2. was previously 0. and this failed.

Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: This 2. was also previously a 0.

Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: I changed these depths

Comment (kbarnhart) [inline landlab/components/lake_fill/lake_fill_barnes.py]: And I changed this volume. Those were all the changes I made to this file.

**Comment by kbarnhart** [comment]
Other than fake coverage changes, it looks like this passes.

**Comment by SiccarPoint** [comment]
@kbarnhart Your review comments are coming up weird for me here. They all point to lines 1428-1431, when those clearly aren't always the relevant lines...? Could you check them?

Also seems like you've restored a bunch of @gregtucker 's code (restoring `cts_lattice_grain_silo`) - I presume this is all good but might be worth telling him if you haven't?

**Comment by kbarnhart** [comment]
@SiccarPoint you are right... the comments all point to the same lines. That was not what I did originally and did not intend. I make the comments to let you know where I changed the docstring test before I realized I could just point you to the specific commit. 

Probably the easiest thing is to just look at [the commit](https://github.com/landlab/landlab/pull/897/commits/525a69f19643ebbce600899757294c1cb1641099) instead of the weird comments... Sorry about that. 

Comment forthcoming about the ctsf files.

**Comment by kbarnhart** [comment]
@SiccarPoint I think what you are seeing is that when @mcflugen made his original move examples PR #835 some of the files moved into the examples folder showed up as the creation of new files rather than the moving of existing files. 

When I look at this PR, almost all the files moved into the examples folder show as direct moves, a few show a small number of lines changed, and a few show as a new file. cts_lattice_grain_silo.py is in this last category along with drive_pot_fr.py, example_point_load.py, and perterb_NMG2_pureDLTL.py.

I've checked it against the file changes in the original PR (#835) and it looks the same to me. 

Does that make sense? Or are you seeing something else that I haven't noticed.

**Comment by SiccarPoint** [comment]
As long as you're happy with it!

**Comment by kbarnhart** [comment]
@SiccarPoint unless you need this to merge ASAP, I think we should let @mcflugen review this. 
I know he is crushing 🚀 towards the PyMT release, though...

**Comment by SiccarPoint** [comment]
Can sit on this a little longer... thanks guys.

**Comment by SiccarPoint** [comment]
@mcflugen Any movement on a possible review? Student is running a whole set of janky workarounds and it is making me really nervous...

**Comment by kbarnhart** [comment]
@SiccarPoint I've just updated the branch so when @mcflugen looks at it it can be merged.

**Comment by kbarnhart** [comment]
Also. I think that once this is merged, we will want to make a new release so your student can get the uptodate version from conda forge?

**Comment by SiccarPoint** [comment]
Yeah, if we could that would be awesome

**Comment by kbarnhart** [comment]
I made a [PR on the landlab-feedstock](https://github.com/conda-forge/landlab-feedstock/pull/6) to add myself as a maintainer... That way @mcflugen doesn't have to be the only release maintainer. Its not passing and I don't really know why...

**Comment by mcflugen** [comment]
@kbarnhart @SiccarPoint This looks good to me.

This is mostly just moving the example scripts out from under the *landlab* package. @SiccarPoint why does your student need to do workarounds? Are they using the priority queue? I notice that that is in this PR (but I'm not clear why).

**Comment by kbarnhart** [comment]
It also involves putting back the new component Dan made that I deleted...

**Comment by kbarnhart** [comment]
@mcflugen can you make a new release?

**Comment by mcflugen** [comment]
@kbarnhart Sure. *v1.8.0*?

**Comment by kbarnhart** [comment]
@mcflugen I think that v1.8.0 makes sense!

**Comment by mcflugen** [comment]
Excellent. It's building now conda-forge/landlab-feedstock#7.

By the way, your tests were failing on conda-forge/landlab-feedstock#6 because the sha256 didn't match the tarball. I'm not sure how they got out of sync.

**Comment by SiccarPoint** [comment]
Thanks so much for this guys!!

**Pull request merged**